src/core/styles/_constants.scss

```diff
@@ -40,6 +40,9 @@ $color-purple-50: hsl(260deg 47% 96%); // #f4f1fa
 // Blue colors
 $color-mentorship-banner: hsl(198deg 83% 70%); // #73ccf2
 $color-mentorship-header: hsl(198deg 86% 55%); // #2BB4EF
+$color-blue-500: hsl(211deg 100% 50%); // #007bff
+$color-blue-800: hsl(212deg 100% 32%); // #004CA3
+$color-blue-background-100: hsl(212deg 100% calc(45% + 49%) / 100%); // #e0efff
```

Comment:
why calc?

Reply:
I just copied these styles from somewhere. I'll try to do the same without calc